moirae/estimators/online/filters/conversions.py

```diff
@@ -168,8 +175,111 @@ def inverse_transform_samples(self, transformed_samples: np.ndarray) -> np.ndarr
         samples = np.matmul(samples, self.inv_multi)
         return samples
 
-    def inverse_transform_covariance(self, transformed_covariance: np.ndarray) -> np.ndarray:
+    def inverse_transform_covariance(self,
+                                     transformed_covariance: np.ndarray,
+                                     transformed_pivot: Optional[np.ndarray] = None) -> np.ndarray:
         if self._len_multi_shape == 0:
             return self.inv_multi * self.inv_multi * transformed_covariance
         covariance = np.matmul(np.matmul(self.inv_multi.T, transformed_covariance), self.inv_multi)
         return covariance
+
+
+class FirstOrderTaylorConversionOperator(ConversionOperator):
+    """
+    Base class that specifies the necessary machinery to perform non-linear conversions assuming a first order Taylor
+    expansion around a pivot point is sufficient to propagate uncertainties (through covariances).
+
+    Assumes the transformation can be expressed as :math:`f = f_0 + J (x-p)`, where :math:`f_0` represents the value of
```

Comment:
How about linking to the Wikipedia page which explains propagation of uncertainty for nonlinear functions: https://en.wikipedia.org/wiki/Propagation_of_uncertainty#Non-linear_combinations? 

FYI: The syntax for hyperlinks in RST is ``` `text <https://link.com>`_ ```

Reply:
Adding the link, but I do like keeping the math explanation here, unless there is a specific documentation or styling issue not to?

Reply:
Good point, let's keep the math in there. The description is simple enough.

Reply:
Put the full explanation at the end, after the summary you write.